spec/services/prompt_augmentor_spec.rb

```diff
@@ -6,32 +6,34 @@
   let(:chunks) do
     [
       create(:chunk),
-      create(:chunk_from_web),
+      # create(:chunk),
     ]
   end
   let(:search_hits) do
     [
       Search::SearchHit.new(chunks[0], 200.0),
-      Search::SearchHit.new(chunks[1], 220.0),
+      # Search::SearchHit.new(chunks[1], 220.0),
     ]
   end
 
+  # TODO: make part of #prompt test below
+  describe "#prompt for doc from web" do
+    it "has the correct content" do
+      c = create(:chunk_from_web)
+      hs = [Search::SearchHit.new(c, 240.0)]
+      p = described_class.new(message, hs)
+      # TODO: Figure out why needed:
+      allow_any_instance_of(Document).to receive(:web?).and_return(true) # rubocop:disable RSpec/AnyInstance
```

Comment:
Oh, I see the `web?` call you added now, but I guess that still doesn't explain why you need to do this.

What was the error you were getting without this line?

Reply:
Without that line the generated prompt has wrong elements (i.e., `<filename>` instead of `<url>` and `<scraped>`).  I think `web?` returns true instead of false because the `document_from_web` and `chunk_from_web` factories I added aren't quite working correctly.

Reply:
It works great if I don't use inheritance, and it may be a little easier to understand, so I think I'll do this:

```ruby
FactoryBot.define do
  factory :chunk do
    document { association(:document) }
    vector_id { SecureRandom.uuid }
    content { Faker::Lorem.sentence(word_count: 10) }
    embedding_content { Faker::Lorem.sentence(word_count: 16) }
  end
  factory :chunk_from_web, class: 'Chunk' do
    document { association(:document_from_web) }
    vector_id { SecureRandom.uuid }
    content { Faker::Lorem.sentence(word_count: 10) }
    embedding_content { Faker::Lorem.sentence(word_count: 16) }
  end
end